relay-metrics/src/aggregation.rs

```diff
@@ -841,6 +847,23 @@ pub struct AggregatorConfig {
     ///
     /// Defaults to `200` bytes.
     pub max_tag_value_length: usize,
+
+    /// Maximum amount of bytes used for metrics aggregation.
+    ///
+    /// When aggregating metrics, Relay keeps track of how many bytes a metric takes in memory.
+    /// This is only an approximation and does not take into account things such as pre-allocation
+    /// in hashmaps.
+    ///
+    /// Defaults to `None`, i.e. no limit.
+    pub max_total_bucket_bytes: Option<usize>,
```

Comment:
Open thought: These configs would fit equally well under `limits` and `aggregator`. Thoughts?

Reply:
Yes, though we currently expose `aggregator:` as-is in the config. It would mean a refactor to split up AggregatorConfig into multiple existing sections.

Reply:
One thought: If the aggregator ever becomes a stand-alone service that scales independently of processing relays (even if it's just a special relay mode), it might make sense to have all its config in one place.